src/Cache/Cache.php

```diff
@@ -79,7 +79,7 @@ public function toJson()
             'hashes' => $this->hashes,
         ));
 
-        if (false === $json) {
+        if (null === $json || false === $json) {
```

Comment:
Is `json_encode()` really returning `null` in case of failure?

Reply:
Didn't read the docs correctly, updated for a better solution